### PR TITLE
Street mode FR

### PR DESCRIPTION
**Pull request by el00ruobuob**
Street mode strings & help translated to french.
`menu_info` string updated according to `Show Secrets!`

### CODE REVIEW

**Comment by el00ruobuob** [comment]
Ready for review @erciccione

**Comment by erciccione** [comment]
Thanks! @assylias care to give this a review?

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -355,9 +355,9 @@
...
 
-    <string name="status_wallet_connect_wrongversion">Node version incompatible - please upgrade!</string>
+    <string name="status_wallet_connect_wrongversion">Version du Nœud incompatible - merci de mettre à jour !</string>
 
-    <string name="menu_info">Détails</string><!--Changed to: Show Secrets!-->
-    <string name="menu_streetmode">Street Mode</string>
-    <string name="info_streetmode_enabled">Street Mode enabled, tap for more info.</string>
+    <string name="menu_info">Montrer les secrets !</string><!--Changed to: Show Secrets!-->
+    <string name="menu_streetmode">Mode Urbain</string>
+    <string name="info_streetmode_enabled">Mode Urbain activé, tapez pour plus d'infos.</string>
```
Comment (erciccione) [inline app/src/main/res/values-fr/strings.xml]: You forgot to escape `'` here

**Review by erciccione — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-fr/help.xml [hunk 2]
```diff
@@ -231,11 +231,12 @@
...
         <h1>Le Portefeuille</h1>
-        <h2>Street Mode</h2>
-        <p>Street mode can be enabled/disabled in the menu or Gunther\'s head icon. In this mode, your
-        balance is not shown on any screen so you can safely use your wallet on the street, a pub or
-        other public place. Previous transactions are also hidden. New transactions will be shown, so
-        you can see that you have sent/received sweet Moneroj!</p>
+        <h2>Mode Urbain</h2>
+        <p>Le mode urbain peut être activé / désactivé dans le menu ou l’icône de la tête de Gunther. Dans
+        ce mode, votre solde n’est visible sur aucun écran afin que vous puissiez utiliser votre
+        portefeuille en toute sécurité dans la rue, un bar ou autre lieu publique. Les transactions
```
Comment (assylias) [inline app/src/main/res/values-fr/help.xml]: publique -> public

File: app/src/main/res/values-fr/help.xml [hunk 3]
```diff
@@ -231,11 +231,12 @@
...
+        précédentes sont également masquées. Les nouvelles transactions seront affichées, afin que vous
+        puissiez voir que vous avez envoyé ou reçus vos doux Moneroj !</p>
```
Comment (assylias) [inline app/src/main/res/values-fr/help.xml]: reçus -> reçu

File: app/src/main/res/values-fr/help.xml [hunk 4]
```diff
@@ -231,11 +231,12 @@
...
         <h2>Analyse</h2>
```
Comment (assylias) [inline app/src/main/res/values-fr/help.xml:240]: A few typos below from a previous commit:

"pour voir si de nouveau Moneroj" -> nouveaux  
"cela ne stock sur votre appareil" -> stocke  
"mon portefeuille à disparu" -> a (no accent)  
"lorsque vous envoyez des fonds" -> Lorsque (upper case)  
"Aprenez-en plus à propos" -> Apprenez

**Review by assylias — COMMENTED** [review]
Just a few comments.

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (el00ruobuob) [reply app/src/main/res/values-fr/strings.xml]: done

File: app/src/main/res/values-fr/help.xml (hunk 2 above)
Comment (el00ruobuob) [reply app/src/main/res/values-fr/help.xml]: done

File: app/src/main/res/values-fr/help.xml (hunk 3 above)
Comment (el00ruobuob) [reply app/src/main/res/values-fr/help.xml]: done

File: app/src/main/res/values-fr/help.xml (hunk 4 above)
Comment (el00ruobuob) [reply app/src/main/res/values-fr/help.xml:240]: done

**Comment by el00ruobuob** [comment]
Thanks @assylias. Updates done.

**Review by erciccione — APPROVED** [review]

**Review by m2049r — APPROVED** [review]
thanks!

**Pull request merged**